### ISSUE TITLE
Karte: Symbole auch bei Populationen

### ISSUE THREAD

**Issue by rebeccakurz** [barbalex/apf2#586]
Könnte in der Karte auch bei den Populationen (wie bei den Teilpopulationen) die Option auswählbar gemacht werden, dass man die Populationen "nach Status, mit Symbolen" anzeigen lassen kann?  
![Bildschirmfoto 2023-01-11 um 14 20 38](https://user-images.githubusercontent.com/30288503/211816984-4bf3cb51-3934-4f9e-ae31-da60c3b31752.png)
![Bildschirmfoto 2023-01-11 um 14 21 05](https://user-images.githubusercontent.com/30288503/211817003-bf7f2403-56c9-4d4f-a21d-1963e9c58a15.png)

**Comment by barbalex** [comment]
Klar. Ist wenige Stunden Arbeit

**Comment by rebeccakurz** [comment]
Super, dann können wir das ja auch für das nächste Budget vorsehen.

**Comment by rebeccakurz** [comment]
inkl. Legende bei der Dokumentation (auch für TPop)

**Comment by rebeccakurz** [comment]
Ja, das wäre super, wenn diese auch im Navigationsbaum eingeblendet werden könnten.

**Comment by barbalex** [comment]
In Version 1.86.0 implementiert
